### PR TITLE
Remove metaparams from cors::configure

### PR DESCRIPTION
**Pull request by alechenninger**

### CODE REVIEW

**Comment by alechenninger** [comment]
Apparently this is a bad thing to do (syntax checker caught this)

**Comment by jewzaam** [comment]
That was the one thing in #83 I wondered about but couldn't find docs around it so just merged.  It's running fine so far, but it makes sense to remove it.

**Comment by dcrissman** [comment]
I found some documentation about puppet's "<a href="https://docs.puppetlabs.com/references/latest/metaparameter.html">metaparameter references</a>", which includes before, require, etc.

File: manifests/service/cors/configure.pp [hunk 1]
```diff
@@ -81,9 +78,6 @@
         owner   => $owner,
         group   => $group,
         content => template('lightblue/properties/lightblue-cors.json.erb'),
-        before  => $before,
-        notify  => $notify,
-        require => $require,
```
Comment (dcrissman) [inline manifests/service/cors/configure.pp]: I am verifying now, but I think you can go ahead and pass these in here to establish the relationships even though they are not defined above.

Comment (alechenninger) [inline manifests/service/cors/configure.pp]: Oh, nice! I'll go ahead and make that change today unless you find out it doesn't work. (I'll poke around that documentation too)

Comment (dcrissman) [inline manifests/service/cors/configure.pp]: Yes, over in #87 I updated the configure.pp and rspec tests. So now the class no longer defines the parameters, but the File definition is still able to use them, as proven by the new rspec test.

I am still looking for what the best practices are for this. I removed $before altogether, as the define should be able to handle that (and presumably $require). But I am still passing $notify to the File.

Comment (alechenninger) [inline manifests/service/cors/configure.pp]: Ah, that makes sense. Thanks for the update.

Comment (jewzaam) [inline manifests/service/cors/configure.pp]: Is it necessary to pass it through to the File?

Comment (dcrissman) [inline manifests/service/cors/configure.pp]: I don't know, but I asked the question over at http://ask.puppetlabs.com/question/15656/what-are-the-best-practices-for-passing-metaparameters-around/

**Pull request merged**